sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/BatchAsyncClient.java

```diff
@@ -10776,17 +12260,12 @@ public PagedFlux<BatchApplication> listApplications() {
      */
     @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<BatchApplication> getApplication(
-            String applicationId, Integer timeOutInSeconds, OffsetDateTime ocpDate) {
+    public Mono<BatchApplication> getApplication(String applicationId, Integer timeOutInSeconds) {
```

Comment:
timeOutInSeconds should probably be in the same kind of options bag object here since it is for list calls.